js-app.js

```diff
@@ -0,0 +1,167 @@
+const menuBar = document.querySelector('#menu-bar');
```

Comment:
As the previous reviewer requested, please make sure your projects section is generated dynamically. Currently, the middle section is empty.  Please test if you project works as expected before submitting for review.

![Screen Recording 2023-04-22 at 6 27 08 PM](https://user-images.githubusercontent.com/25887253/233793622-3ae64cb0-a698-4173-ac61-3cdd10d65776.gif)